lib/client-logger/package.json

```diff
@@ -15,7 +15,7 @@
   },
   "license": "MIT",
   "main": "dist/index.js",
-  "jsnext:main": "src/index.js",
+  "types": "src/index.d.ts",
```

Comment:
This should be `dist/index.d.ts` I think?

Reply:
Fixed this – thanks for pointing it out.